interfaces/naming.go

```diff
@@ -32,3 +32,7 @@ func SecurityTagGlob(snapName string) string {
 func InterfaceServiceName(snapName, uniqueName string) string {
 	return snap.ScopedSecurityTag(snapName, "interface", uniqueName) + ".service"
 }
+
+func InterfacePolkitPolicyName(snapName, uniqueName string) string {
+	return snap.ScopedSecurityTag(snapName, "interface", uniqueName) + ".policy"
```

Comment:
this means we will have usually one policy file per snap? does it make sense to have policy file per app instead?

Reply:
The backend is agnostic about how many policy files a snap will install. The interface I've built on top of it currently only supports one .policy file per connected plug. It looks like there are a few cases of apps installing multiple policy files on my system:

* gnome-control-center has three (datetime, remote-login-helper, and user-accounts).
* systemd has seven (hostname1, locale1, login1, network1, resolve1, systemd1, and timedate1).

While in the systemd case these correspond to different system daemons, in the case of gnome-control-center they relate to different pages of the settings app (i.e. all the one process).  So I don't think it is something that really maps to snap apps.

As far as polkitd is concerned, there's no benefit to separating actions into multiple files (it allows actions to inherit some metadata defaults from the top level of file, but that's about it). But it would be good to avoid snap packagers from having to edit or merge upstream policy files.